package.json

```diff
@@ -48,7 +48,7 @@
     "hs-client": "~0.0.9",
     "n64": "~0.2.10",
     "mrmr": "~0.1.8",
-    "namebase-hs-client": "=0.0.5",
+    "namebase-hs-client": "github:namebasehq/hs-client#master",
```

Comment:
Can you make sure this is pinned to a specific version and not to the master branch? Otherwise we won't be able to control what version of namebase-hs-client the different deployments of hsd ship with. Make sure to delete namebase-hs-client from the package-lock file when you do this so that it updates properly with the pinned version.

Reply:
done